### ISSUE TITLE
Release 1.2.1 Is a breaking change

### ISSUE THREAD

**Issue by gwshaw** [fastly/fastly-ruby#65]
After updating to 1.2.1 from 1.2.0, I get the following error when using the Fastly Gem to install Fastly code:

OpenSSL::SSL::SSLError:
       SSL_connect returned=1 errno=0 state=SSLv3 read server certificate B: certificate verify failed

**Comment by lanej** [comment]
@gwshaw can you give us a full backtrace with your initialization parameters ?

**Comment by ahacop** [comment]
I have the same issue with 1.2.1. Downgrading to 1.2.0 works. Let me know if any other info would be helpful. Thanks!

Backtrace:
```
OpenSSL::SSL::SSLError: SSL_connect returned=1 errno=0 state=SSLv3 read server certificate B: certificate verify failed
... 60 non-project frames 
1
File "/app/vendor/ruby-2.2.4/lib/ruby/2.2.0/net/http.rb" line 923 in connect
2
File "/app/vendor/ruby-2.2.4/lib/ruby/2.2.0/net/http.rb" line 923 in block in connect
3
File "/app/vendor/ruby-2.2.4/lib/ruby/2.2.0/timeout.rb" line 73 in timeout
4
File "/app/vendor/ruby-2.2.4/lib/ruby/2.2.0/net/http.rb" line 923 in connect
5
File "/app/vendor/ruby-2.2.4/lib/ruby/2.2.0/net/http.rb" line 1457 in begin_transport
6
File "/app/vendor/ruby-2.2.4/lib/ruby/2.2.0/net/http.rb" line 1410 in transport_request
7
File "/app/vendor/ruby-2.2.4/lib/ruby/2.2.0/net/http.rb" line 1384 in request
8
File "/app/vendor/bundle/ruby/2.2.0/gems/skylight-0.10.3/lib/skylight/probes/net_http.rb" line 34 in block in request
9
File "/app/vendor/bundle/ruby/2.2.0/gems/skylight-0.10.3/lib/skylight/instrumenter.rb" line 219 in instrument
10
File "/app/vendor/bundle/ruby/2.2.0/gems/skylight-0.10.3/lib/skylight/core.rb" line 108 in instrument
11
File "/app/vendor/bundle/ruby/2.2.0/gems/skylight-0.10.3/lib/skylight/probes/net_http.rb" line 33 in request
12
File "/app/vendor/ruby-2.2.4/lib/ruby/2.2.0/net/http.rb" line 1398 in send_entity
13
File "/app/vendor/ruby-2.2.4/lib/ruby/2.2.0/net/http.rb" line 1186 in post
14
File "/app/vendor/bundle/ruby/2.2.0/gems/fastly-1.2.1/lib/fastly/client.rb" line 95 in post_and_put
15
File "/app/vendor/bundle/ruby/2.2.0/gems/fastly-1.2.1/lib/fastly/client.rb" line 79 in post
16
File "/app/vendor/bundle/ruby/2.2.0/gems/fastly-rails-0.4.1/lib/fastly-rails/client.rb" line 12 in purge_by_key
17
File "/app/vendor/bundle/ruby/2.2.0/gems/fastly-rails-0.4.1/lib/fastly-rails.rb" line 26 in purge_by_key
18
File "/app/vendor/bundle/ruby/2.2.0/gems/fastly-rails-0.4.1/lib/fastly-rails/active_record/surrogate_key.rb" line 33 in purge
19
File "/app/vendor/bundle/ruby/2.2.0/gems/activesupport-4.2.5.2/lib/active_support/callbacks.rb" line 432 in block in make_lambda
20
File "/app/vendor/bundle/ruby/2.2.0/gems/activesupport-4.2.5.2/lib/active_support/callbacks.rb" line 228 in call
21
File "/app/vendor/bundle/ruby/2.2.0/gems/activesupport-4.2.5.2/lib/active_support/callbacks.rb" line 228 in block in halting_and_conditional
22
File "/app/vendor/bundle/ruby/2.2.0/gems/activesupport-4.2.5.2/lib/active_support/callbacks.rb" line 506 in call
23
File "/app/vendor/bundle/ruby/2.2.0/gems/activesupport-4.2.5.2/lib/active_support/callbacks.rb" line 506 in block in call
24
File "/app/vendor/bundle/ruby/2.2.0/gems/activesupport-4.2.5.2/lib/active_support/callbacks.rb" line 506 in each
25
File "/app/vendor/bundle/ruby/2.2.0/gems/activesupport-4.2.5.2/lib/active_support/callbacks.rb" line 506 in call
26
File "/app/vendor/bundle/ruby/2.2.0/gems/activesupport-4.2.5.2/lib/active_support/callbacks.rb" line 92 in __run_callbacks__
27
File "/app/vendor/bundle/ruby/2.2.0/gems/activesupport-4.2.5.2/lib/active_support/callbacks.rb" line 778 in _run_save_callbacks
28
File "/app/vendor/bundle/ruby/2.2.0/gems/activerecord-4.2.5.2/lib/active_record/callbacks.rb" line 302 in create_or_update
29
File "/app/vendor/bundle/ruby/2.2.0/gems/activerecord-4.2.5.2/lib/active_record/persistence.rb" line 120 in save
30
File "/app/vendor/bundle/ruby/2.2.0/gems/activerecord-4.2.5.2/lib/active_record/validations.rb" line 37 in save
31
File "/app/vendor/bundle/ruby/2.2.0/gems/activerecord-4.2.5.2/lib/active_record/attribute_methods/dirty.rb" line 21 in save
32
File "/app/vendor/bundle/ruby/2.2.0/gems/activerecord-4.2.5.2/lib/active_record/transactions.rb" line 286 in block (2 levels) in save
33
File "/app/vendor/bundle/ruby/2.2.0/gems/activerecord-4.2.5.2/lib/active_record/transactions.rb" line 351 in block in with_transaction_returning_status
34
File "/app/vendor/bundle/ruby/2.2.0/gems/activerecord-4.2.5.2/lib/active_record/connection_adapters/abstract/database_statements.rb" line 211 in transaction
35
File "/app/vendor/bundle/ruby/2.2.0/gems/activerecord-4.2.5.2/lib/active_record/transactions.rb" line 220 in transaction
36
File "/app/vendor/bundle/ruby/2.2.0/gems/activerecord-4.2.5.2/lib/active_record/transactions.rb" line 348 in with_transaction_returning_status
37
File "/app/vendor/bundle/ruby/2.2.0/gems/activerecord-4.2.5.2/lib/active_record/transactions.rb" line 286 in block in save
38
File "/app/vendor/bundle/ruby/2.2.0/gems/activerecord-4.2.5.2/lib/active_record/transactions.rb" line 301 in rollback_active_record_state!
39
File "/app/vendor/bundle/ruby/2.2.0/gems/activerecord-4.2.5.2/lib/active_record/transactions.rb" line 285 in save
```

Initialization:
```
# config/initializers/fastly.rb
FastlyRails.configure do |c|
  c.api_key = ENV['FASTLY_API_KEY']  # Fastly api key, required
  #c.max_age = 86400                  # time in seconds, optional, defaults to 2592000 (30 days)
  c.service_id = ENV['FASTLY_SERVICE_ID']   # The Fastly service you will be using, required
  c.purging_enabled = !Rails.env.development? # No need to configure a client locally (AVAILABLE ONLY AS OF 0.4.0)
end
```

**Comment by lanej** [comment]
This issue should be fixed by #71. Which is released in https://rubygems.org/gems/fastly/versions/1.2.2.

**Comment by ahacop** [comment]
Seems to have cleared up my issue. Thanks! :bow:
